first.js

```diff
@@ -5,6 +5,11 @@ var pwrlvl = function () {
     if (result > 9000) {
         result += " IT'S OVER 9000!!!!!";
     }
+
+    if (result < 2000) {
+        result += "HEUGHHHHH";
```

Comment:
Can we change `HEUGHHHHH` to `HEUGHHHHHH`?

Reply:
Good contributions